arbiter/server_test.go

```diff
@@ -407,7 +407,18 @@ func (s *syncBinlogsSuite) TestShouldSendBinlogToLoader(c *C) {
 	}()
 	ld := dummyLoader{input: dest}
 
-	syncBinlogs(source, &ld)
+	res := make(chan error)
+	var err error = nil
```

Comment:
Note that Go initialize all variables to the empty value of its type, so it's OK to just use `var err error` here.